examples/PathHandler.js

```diff
@@ -60,9 +60,10 @@ PathHandler.prototype.addShape = function(shape) {
  *  @param {Number} x
  *  @param {Number} y
  */
-PathHandler.prototype.arcAbs = function(rx, ry, xAxisRotation, largeArcFlag, sweepFlag, x, y) {
+PathHandler.prototype.arcAbs = function(/* rx, ry, xAxisRotation, largeArcFlag, sweepFlag, x, y */) {
```

Comment:
I'm assuming this is being commented out for eslint? If so, it feels a little strange to have docs on arguments that have been commented out. Is there another way to tell eslint to ignore the function signature?

Reply:
Right, for ESLint. I've added back the args but with the `eslint-disable-next-line` directive just above this line (I could use `eslint-disable-line` directive on the same line instead, but besides potentially interfering with the `max-len` rule if you ever wanted that, it can get harder to notice such comments and thus maintain them.)

Reply:
LGTM